CharlieBackend.Business/Services/Interfaces/IStudentService.cs

```diff
@@ -9,20 +9,22 @@ public interface IStudentService
     {
         Task<Result<StudentDto>> CreateStudentAsync(long accountId);
 
-        Task<IList<StudentDto>> GetAllStudentsAsync();
+        Task<Result<IList<StudentDto>>> GetAllStudentsAsync();
 
-        Task<IList<StudentDto>> GetAllActiveStudentsAsync();
+        Task<Result<IList<StudentDto>>> GetAllActiveStudentsAsync();
 
         Task<Result<IList<StudentStudyGroupsDto>>> GetStudentStudyGroupsByStudentIdAsync(long id);
 
         Task<long?> GetAccountId(long studentId);
 
         Task<Result<StudentDto>> UpdateStudentAsync(long id, UpdateStudentDto studentModel);
 
-        Task<StudentDto> GetStudentByAccountIdAsync(long accountId);
+        Task<Result<StudentDto>> GetStudentByAccountIdAsync(long accountId);
 
-        Task<StudentDto> GetStudentByIdAsync(long studentId);
+        Task<Result<StudentDto>> GetStudentByIdAsync(long studentId);
```

Comment:
Don't forget to call .ToActionResult in controller action. Otherwise, it will change schema of response body.